source/tit/core/par/control.cpp

```
@@ -30,8 +30,8 @@ void set_num_threads(size_t value) {
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-auto global_mutex() noexcept -> std::mutex& {
-  static std::mutex mutex{};
+auto global_mutex() noexcept -> std::recursive_mutex& {
```

Cppcheck (2.10-2)

[style] The function 'global_mutex' is never used.

(unusedFunction)